README.md

```diff
@@ -48,9 +55,8 @@ Using Ruby, remove Belle from the town residents, and
 add her to the list of guests in the castle.
 
 Write your code here:
-```ruby
-# code here
-```
+
+residents.index(1).push
```

Comment:
First thing to do here is make sure we're accessing the `residents` array properly. We can begin our code with `residents` because it doesn't exist in our global scope. We can, however, begin with `town` and then navigate throw it's key-value pairs to access `residents`. So we'd do something like...

```rb
# This takes us to the residents array. What do we do next?
town[:residents]
```

Next thing to consider is how do we access an array value in Ruby. You used an `.index` method, but it's simpler than that. We access array values the same way we do in Javascript...

Lastly, you used `.push` to try and remove "Belle" from the `residents` array. But `.push` does the opposite -- it adds something to the end of an array. Instead, you're going to want to use `.delete`...